### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by 0xInfection** [s0md3v/Arjun#167]
pls fix
```
Traceback (most recent call last):
  File "/usr/local/bin/arjun", line 33, in <module>
    sys.exit(load_entry_point('arjun==2.1.5', 'console_scripts', 'arjun')())
  File "/usr/local/lib/python3.8/dist-packages/arjun/__main__.py", line 160, in main
    these_params = initialize(request, wordlist)
  File "/usr/local/lib/python3.8/dist-packages/arjun/__main__.py", line 134, in initialize
    param_groups = narrower(request, factors, param_groups)
  File "/usr/local/lib/python3.8/dist-packages/arjun/__main__.py", line 94, in narrower
    if result.result():
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.8/dist-packages/arjun/core/bruter.py", line 21, in bruter
    comparison_result = compare(response, factors, params)
  File "/usr/local/lib/python3.8/dist-packages/arjun/core/anomaly.py", line 74, in compare
    if value in response.text and re.search(r'[\'"\s]%s[\'"\s]' % value, response.text):
TypeError: 'in <string>' requires string as left operand, not list
```
pls fix

**Comment by pacho15** [comment]
i m getting same error

**Comment by s0md3v** [comment]
This issue has been fixed in Arjun 2.1.6 🎉

Thanks for reporting!
